cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -69,6 +69,15 @@ public void updateUser(final MetricUser user) {
             logger.debug("Error updating a Intercom user: " + intercomUser + " - " + e.getMessage());
         }
     }
+    
+    @Override
+    public void deleteUser(final String userId) {
+    	try {
```

Comment:
Why not following same pattern as other methods in this class and pass in a MetricUser?